src/delta.rs

```diff
@@ -509,7 +517,7 @@ fn handle_hunk_header_line(
                 &painter.output_buffer,
                 &painter.output_buffer,
                 &config.decorations_width,
-                config.hunk_header_style,
+                config.null_style,
```

Comment:
This change is orthogonal to this PR. Basically we were painting the hunk header text twice, since the text being passed to `draw_fn` has already been painted. So I believe it's a bug fix, but not a bug likely to have been noticed by anyone. It caused the unit test here to fail, because I was expecting the line to start with `file-style`, but in fact it started with `hunk-header-style`. I.e. the line looked like

```
(hunk-header-style) (file-style) file.rs (hunk-header-style) fn my_fun () {
```
and with this fix it looks like
```
(file-style) file.rs (hunk-header-style)  fn my_fun () {
```
The commit message:
```
commit 31066fc8803e47e5f992c53ab611e8077093693a
Author: Dan Davison <dandavison7@gmail.com>
Date:   Sat Dec 19 12:32:30 2020 +0000

    Do not paint hunk header twice
    
    The text being passed to the draw function has already been painted.

 src/delta.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
```